### ISSUE TITLE
Pushing multiple test executions when executing tests in parallel using pytest-xdist

### ISSUE THREAD

**Issue by RoCCoCo13** [fundakol/pytest-jira-xray#74]
Hi there,

I have identified that whenever I execute test cases using the plugin `pytest-xdist`, more than one x-ray test execution will be sent to JIRA.

One test execution for each worker and one at the end with all the results together.

In the logs I can see only a message of one test execution created, but actually it creates more:
```
 Uploaded results to JIRA XRAY. Test Execution Id: VSDS-19800
```
The ticket number shown is actually the last execution created.

Here are the versions I use:
```
$ python --version
Python 3.10.0
$ pip freeze | grep pytest
pytest==7.2.0
pytest-cov==3.0.0
pytest-html==3.1.1
pytest-jira-xray==0.8.6
pytest-metadata==2.0.1
pytest-xdist==3.3.1
```

Does anyone know if there is a specific configuration needed to avoid this behaviour? I couldn't find anything related to that in the documentation.

Many thanks

**Comment by fundakol** [comment]
@RoCCoCo13 thank you for reporting this issue. I fixed it in 0.8.7.

**Comment by RoCCoCo13** [comment]
Hi @fundakol, tested and works like a charm. Thanks!
